aspnetcore/blazor/get-started.md

```diff
@@ -30,6 +135,8 @@ Get started with Blazor:
 
    1\. Install the latest [Visual Studio](https://visualstudio.com/vs/) with the **ASP.NET and web development** workload.
 
+       Optionally install [Visual Studio 16.4 Preview 2 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload for Blazor WebAssembly app development.
```

Comment:
For some reason this is rendering as a code block on the review site:

![image](https://user-images.githubusercontent.com/1874516/66872832-b9d02880-ef5b-11e9-9c58-cfea70a2868c.png)

Reply:
I'll take a look.

Reply:
I'm going to give that a dedicated list number ... that hopefully will fix it. 🤞 Stand-by ....